docs/en/query_language/select.md

```diff
@@ -448,6 +448,19 @@ The table names can be specified instead of `<left_subquery>` and `<right_subque
 
 See the standard [SQL JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) description.
 
+#### Syntax Limitations
+
+For multiple `JOIN`:
+
+- You cannot perform `SELECT * ...` if you use `SELECT` subqueries for joining. Using of `*` is available only for joining tables.
+- The `PREWHERE` clause is not available for multiple `JOIN`.
+
+For `ON`, `WHERE` and `GROUP BY` clauses:
+
+- You cannot use the functions in clauses.
+
+    You can use the functions in the `SELECT` list, define an aliases for them and use aliases in `ON`, `WHERE` and `GROUP BY` clauses.
+
 #### Multiple JOIN
 
 Performing queries, ClickHouse rewrites multiple joins into the combination of two-table joins and processes them sequentially. If there are four tables for join ClickHouse joins the first and the second, then joins the result with the third table, and at the last step, it joins the fourth one.
```

Comment:
combination ... and processes them sequentially -> just "sequence of two-table joins"